.editorconfig

```diff
@@ -9,9 +9,13 @@ insert_final_newline = true
 indent_style = space
 indent_size = 4
 trim_trailing_whitespace = true
+max_line_length = 80
```

Comment:
I am not sure, whether this is actually necessary when writing text …

Reply:
Ah. Yeah. Not too important. Personal preference because then editors show this line on the right.

![image](https://github.com/TUD-OS/latex-template/assets/5737016/9cc83bac-7c63-4b1b-b108-36b13714c670)